testing/src/runner/index.ts

```diff
@@ -9,10 +10,11 @@ export const runTest = async (
     cookies: {},
     envVars: env || {},
   }
+
   if (test.env) {
     context.envVars = Object.fromEntries(
       Object.entries(context.envVars).concat(
-        test.env.map(e => [e.name, e.value]),
+        test.env.map(e => [e.name, Handlebars.compile(e.value)(env)]),
```

Comment:
can you do this one at a time so env items can reference previously declared stuff